.github/renovate.json

```diff
@@ -3,5 +3,11 @@
   "extends": [
     "local>hmcts/.github:renovate-config",
     "local>hmcts/.github//renovate/automerge-minor"
+  ],
+  "timezone": "Europe/London",
+  "schedule": [
+    "after 6pm every weekday",
+    "before 8am every weekday",
```

Comment:
The platform gets turned off every night so might want to tighten this to something like "before 8pm"?